app/views/sprint_feedbacks/edit_retro.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { PageProps } from '../../../data.d';
+import { useTranslate } from '../../frontend/util/dependencies';
+import { Stack } from '../../frontend/components/stack/stack';
+import { Button } from '../../frontend/components/button/button';
+import { useForm } from '@nerdgeschoss/react-use-form-library';
+import { TextField } from '../../frontend/components/text_field/text_field';
+import { useReaction } from '../../frontend/sprinkles/reaction';
+import { useModalInfo } from '../../frontend/components/modal/modal';
+import { NumberField } from '../../frontend/components/number_field/number_field';
+import { Checkbox } from '../../frontend/components/checkbox/checkbox';
+
+export default function ({
+  data: { feedback },
+}: PageProps<'sprint_feedbacks/edit_retro'>): JSX.Element {
+  const t = useTranslate();
+  const reaction = useReaction();
+  const modal = useModalInfo();
+  const { fields, valid, onSubmit } = useForm({
+    model: {
+      retroRating: feedback.retroRating ?? 5,
+      retroText: feedback.retroText ?? '',
+      skipRetro: feedback.skipRetro,
+    },
+    validations: {
+      retroRating: ({ model }) =>
+        model.skipRetro || model.retroRating ? [] : ['required'],
+      retroText: ({ model }) =>
+        model.skipRetro || model.retroText ? [] : ['required'],
+    },
+    onSubmit: async ({ model }) => {
+      const sprint_feedback = model.skipRetro
+        ? { skipRetro: true, retroText: null, retroRating: null }
+        : {
+            skipRetro: false,
+            retroText: model.retroText,
+            retroRating: model.retroRating,
+          };
+      await reaction.call({
+        path: `/sprint_feedbacks/${feedback.id}/update_retro`,
```

Comment:
I don't know where best to put this comment, but when I submit this form with invalid values (like 500 for the rating), then the server response is the Rails Error page as HTML. This is not handled by Reaction and no error message at all is ever displayed. (If I did't look at the network inspector, I wouldn't know an error happened.)

<img width="802" alt="image" src="https://github.com/user-attachments/assets/619893d7-28fc-4f19-b6a1-4192a1f88ef4" />

Reply:
I added some basic error handling